docs/faq/env_var.md

```diff
@@ -126,6 +126,11 @@ When USE_PROFILER is enabled in Makefile or CMake, the following environments ca
   - Values: String ```(default='https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/'```
   - The repository url to be used for Gluon datasets and pre-trained models.
 
+* MXNET_DATA_DIR
```

Comment:
can you elaborate? I is this path hardcoded somewhere else?

Reply:
Apologies for the confusion. What I meant is we can use a variable name such as MXNET_HOME so that other assets such as models and datasets can all be saved there.

Reply:
okay, I don't see what is the different semantic with dada dir and home, might be my poor english skills, so fine to take the suggestion.